data-service/src/main/java/uk/gov/gchq/palisade/service/data/service/ExceptionHandler.java

```diff
@@ -81,7 +80,7 @@
     private ResponseEntity<Object> noPolicyExceptionHandler(final NoPolicyException ex, final WebRequest request) {
         ErrorDetails details = new ErrorDetails(ZonedDateTime.now(), MESSAGE, ex.getMessage(), ex.getStackTrace());
 
-        LOGGER_DEBUG.error("Error from data service, details: {}, and request {}", details, request);
+        LOGGER.error("Error from data service, details: {}", details);
```

Comment:
It is useful to have the request available that caused the exception
```suggestion
       LOGGER.error("Error from data service, details: {}, and request {}", details, request);
```